src/utils.ts

```diff
@@ -180,10 +274,17 @@ export const getTraceId = (awsXAmznTraceId) => {
 
 export const getPatchedTraceId = (awsXAmznTraceId): string => {
   // @ts-ignore
-  const { Root, Parent, Sampled, transactionId } = getTraceId(awsXAmznTraceId);
+  const { Root, Parent, Sampled, transactionId, Lineage } = getTraceId(awsXAmznTraceId);
   const rootArr = Root.split('-');
   const currentTime = Math.floor(Date.now() / 1000).toString(16);
-  return `Root=${rootArr[0]}-${currentTime}-${transactionId};Parent=${Parent};Sampled=${Sampled}`;
+  let patchedTraceId = `Root=${rootArr[0]}-${currentTime}-${transactionId};Parent=${Parent}`;
+  if (Sampled) {
+    patchedTraceId += `;Sampled=${Sampled}`;
+  }
+  if (Lineage) {
+    patchedTraceId += `;Lineage=${Lineage}`;
+  }
+  return patchedTraceId;
```

Comment:
I think that concatenating here is a bit tricky with the leading `;`, WDYT of this:
```suggestion
  const patchedTraceId = [`Root=${rootArr[0]}-${currentTime}-${transactionId}`, `Parent=${Parent}`]
  if (Sampled) {
    patchedTraceId.push(`Sampled=${Sampled}`);
  }
  if (Lineage) {
    patchedTraceId.push(`Lineage=${Lineage}`);
  }
  return patchedTraceId.join(';');
```
?